docs/permissions.rst

```diff
@@ -0,0 +1,431 @@
+|                |                        | objects in the collection.       |
+|                +------------------------+----------------------------------+
+|                | `read_collection`      | Ability to read all objects in   |
+|                |                        | the collection.                  |
+|                +------------------------+----------------------------------+
+|                | `create_record`        | Ability to create a new record   |
+|                |                        | in the collection.               |
++----------------+------------------------+----------------------------------+
+| ``record``     | `write_record`         |                                  |
+|                |                        |                                  |
+|                +------------------------+----------------------------------+
+|                | `read_record`          |                                  |
+|                |                        |                                  |
++----------------+------------------------+----------------------------------+
+| ``schema``     | `write_schema`         |                                  |
+|                |                        |                                  |
+|                +------------------------+----------------------------------+
+|                | `read_schema`          |                                  |
+|                |                        |                                  |
++----------------+------------------------+----------------------------------+
+| ``group``      | `write_group`          |                                  |
+|                |                        |                                  |
+|                +------------------------+----------------------------------+
+|                | `read_group`           |                                  |
+|                |                        |                                  |
++----------------+------------------------+----------------------------------+
+             
+.. note::
+
+  Anyone with the `write` permission on an object can also edit its associated
+  permissions.
+
+Principals
+==========
+
+XXX Describe how principals are used.
+
+
+Examples
+========
+
+To better understand how this works, here is a handful of examples which expose
+how the permission model works.
+
+The Payments use case
+---------------------
+
+For the payment use case we have three players involved:
+
+- The **payment app**, storing receipts for buyers and sellers;
+- The **selling app**, reading receipts for a given seller.
+- The **buyer app** reading receipts for a given buyer.
+
+Users shouldn't be able to write receipts themselves, sellers and users should
+only be able to read their owns.
+
+In this case, the ``payments`` bucket will be created with the following ACLs:
+
+.. code-block:: json
+
+    {
+        "id": "payments",
+        "acls": {
```

Comment:
acls is not suitable IMO :)

``permissions`` is a lot better, and consistent with buckets